VERSION

```diff
@@ -1 +1 @@
-0.1.10
+0.1.11
```

Comment:
still I prefer to use 0.2.0 so it is visible that it adds new functionality. I also hope that I can release 1.0 quickly, as I want to have stable API.